webapp/views/partials/users.jade

```diff
@@ -1,8 +1,4 @@
-extends ../layout
-
-block content
-    div(class='container')
-        h1 Users page
-
-        p(ng-repeat='user in users') Username: {{user.email}} Password: {{user.password}}
+div(class='container')
+    h1 Welcome
 
+    p(ng-repeat='user in users') Username: {{user.email}} Password: {{user.password}} Firstname: {{user.firstname}} Lastname {{user.lastname}}
```

Comment:
line is too long. Should look like:
p(stuff).
    content1
    content2
    ....